chapters/5-processing.tex

```diff
@@ -100,32 +107,37 @@ \subsection{Unique identifiers}
   reflecting only that information'',
   depending on the context.
 	This applies to data acquired from partners as well as
-	primary data collected by the research team.}
-It is also possible for a raw dataset to not include a unique identifier,
-or for the identifier to involve a long string
-that is difficult to work with, such as a person's name.
-In such cases, cleaning begins by either carefully creating a variable
-that uniquely identifies the data,
-or merging the ID variable from the master dataset
+	original data collected by the research team.
+}
+
+It is also possible for a raw dataset to not include an unique identifier,
+or that the identifier is not a suitable \textbf{project ID}.\sidenote{
+	\textbf{Project ID:} The main ID used in your project to identify
+	observations.
+	You should never have multiple project IDs for the same unit of observation.
+	The project ID must uniquely and fully identify all observations in the project.
+	See \url{https://dimewiki.worldbank.org/ID\_Variable\_Properties} for more details.}
```

Comment:
Why are we repeating the definition here?

Reply:
As it is the first time it is mentioned in this chapter

Reply:
Should it maybe be an explanation but not the full definition?

Reply:
I would think we define only once in book and then can point to wiki in subsequent chapters. Otherwise we will create maintenance problems whenever a definition is updated, needing to do it consistently

Reply:
Agree with Ben, should only define terms once in the book, but linking to a wiki page for later entries makes sense (or mention here that this is discussed in ch3)

Reply:
ok, now in the defintion is only in ch 3, but in both ch 3 and 5 we link to the new Project ID wiki page. Done in: 885c47eabf04f59cb9b637ae55d64f5ea9a5fdf9